### ISSUE TITLE
OrgChart is in conflict with other third party libraries

### ISSUE THREAD

**Issue by notarnet** [BALKANGraph/OrgChartJS#62]
Hi,
I have a problem with other libraries imported in my project, because OrgChart is overriding standard objects. For example:
```javascript
 Array.prototype.move = function(b, c) {
    if (b == undefined || b == null) {
        return false
    }
    if (c == undefined || c == null) {
        return false
    }
    if (b == c) {
        return false
    }
    var a, d;
    b = parseInt(b, 10);
    c = parseInt(c, 10);
    if (b !== c && 0 <= b && b <= this.length && 0 <= c && c <= this.length) {
        d = this[b];
        if (b < c) {
            for (a = b; a < c; a++) {
                this[a] = this[a + 1]
            }
        } else {
            for (a = b; a > c; a--) {
                this[a] = this[a - 1]
            }
        }
        this[c] = d
    }
};
```
After OrgChart is loaded all arrays are overridden. How to avoid this? This is a big problem!

I solved with this code on dispose:
```javascript
if (Array.prototype.move) {
    delete Array.prototype.move;
}
```

**Comment by BALKANGraph** [comment]
The function is removed in the latest version 4.5.0

**Comment by notarnet** [comment]
Solved in 4.5.1
